astra-sim/system/Sys.cc

```diff
@@ -553,6 +554,7 @@ void Sys::try_register_event(
       make_tuple(callable, event, callData));
   if (should_schedule) {
     timespec_t tmp;
+    tmp.time_res = NS;
```

Comment:
Do we still use `time_res`?

Reply:
This I have to double-check, we should consider removing this in the updated ASTRA-sim API as I haven't seen a scenario using different time_res other than NS, anyway.